src/mixer/playermanager.h

```diff
@@ -36,21 +36,20 @@ class PlayerManagerInterface {
     virtual BaseTrackPlayer* getPlayer(const QString& group) const = 0;
     virtual BaseTrackPlayer* getPlayer(const ChannelHandle& channelHandle) const = 0;
 
-    // Get the deck by its deck number. Decks are numbered starting with 1.
-    virtual Deck* getDeck(unsigned int player) const = 0;
+    // Get the deck by its deck index
+    virtual BaseTrackPlayer* getDeckBase(int deckIndex) const = 0;
```

Comment:
Why this replacement?

Reply:
This interface allows the unittest to return a mocked Deck instead of the real one. I will add a comment. Before `getPlayer()` has been used, which requires a QString group for lookup. I will add a comment.